config/initializers/sentry.rb

```diff
@@ -0,0 +1,7 @@
+Sentry.init do |config|
+  config.breadcrumbs_logger = %i[active_support_logger http_logger]
+
+  # To activate performance monitoring, set one of these options.
+  # We recommend adjusting the value in production:
+  config.traces_sample_rate = 0.5
```

Comment:
Do we want to be configuring environments in here? Something like:

```
config.environments = %w[staging production]
config.rails_report_rescued_exceptions = true
```

Reply:
It is not needed, a lot is actually fetched from Rails.
I would leave the default which uses RAILS_ENV.
Happy to add the other one.

Reply:
Sorry that was just a question. Happy to leave the default as well.